main.nf

```diff
@@ -86,9 +87,11 @@ workflow {
     image_ch = GET_SUBMISSIONS.output 
         .splitCsv(header:true) 
         .map { row -> tuple(row.submission_id, row.image_id) }
-    UPDATE_SUBMISSION_STATUS_BEFORE_RUN(image_ch.map { tuple(it[0], "EVALUATION_IN_PROGRESS") })
+    UPDATE_SUBMISSION_STATUS_BEFORE_RUN(image_ch.map { tuple(it[0], "EVALUATION_IN_PROGRESS") }, "ready")
     RUN_DOCKER(image_ch, staged_path, params.cpus, params.memory)
```

Comment:
We can add a ticket, but does there need to be a parameter added for the 

"UPDATE_SUBMISSION_STATUS_BEFORE_RUN" to run before 'RUN_DOCKER'?

Reply:
I think the workflow will work just fine without this, but I agree that logically it makes sense for statuses to be set to `EVALUATION_IN_PROGRESS` before the `RUN_DOCKER` step. I can make the change now.